src/main/java/org/toradocu/extractor/BlockTag.java

```diff
@@ -0,0 +1,165 @@
+   *
+   * @return the kind of this tag
+   */
+  public Kind getKind() {
+    return kind;
+  }
+
+  /**
+   * Returns the comment associated with the exception in this tag.
+   *
+   * @return the comment associated with the exception in this tag
+   */
+  public Comment getComment() {
+    return comment;
+  }
+
+  /**
+   * Sets the comment for this tag.
+   *
+   * @param comment the comment for this tag, must not be null
+   */
+  public void setComment(Comment comment) {
+    Checks.nonNullParameter(comment, "comment");
+    this.comment = comment;
+  }
+
+  /**
+   * Returns the specification that represents the translation for this tag. {@code null} if the
+   * translation has not been attempted yet or no translation has been generated.
+   *
+   * @return the translation for this tag. {@code null} if the translation has not been attempted
+   *     yet or no translation has been generated.
+   */
+  public S getSpecification() {
+    return specification;
+  }
+
+  /**
+   * Sets the specification (translation) for this tag to the given specification.
+   *
+   * @param specification the comment translation for this tag (a specification)
+   */
+  public void setSpecification(S specification) {
+    this.specification = specification;
+  }
+
+  /**
+   * Returns true if this {@code BlockTag} and the specified object are equal.
+   *
+   * @param obj the object to test for equality
+   * @return true if this object and {@code obj} are equal
+   */
+  @Override
+  public boolean equals(Object obj) {
+    if (this == obj) return true;
+    if (!(obj instanceof BlockTag)) return false;
+
+    BlockTag that = (BlockTag) obj;
+    return kind.equals(that.kind)
+        && comment.equals(that.comment)
+        && Objects.equals(specification, that.specification);
+  }
+
+  /**
+   * Returns the hash code of this object.
+   *
+   * @return the hash code of this object
+   */
+  @Override
+  public int hashCode() {
+    return Objects.hash(comment, kind, specification);
+  }
+
+  /**
+   * Returns a string representation of this tag. The returned string is in the format "@throws
+   * COMMENT" where COMMENT is the text of the comment in the tag. If translation has been attempted
+   * on this tag, then the returned string is also appended with " ==&gt; SPECIFICATION" where
+   * CONDITION is the translation of this tag.
+   *
+   * @return a string representation of this tag
+   */
+  @Override
+  public String toString() {
+    String result = kind + " " + comment.getText();
+    return appendCondition(result);
+  }
+
+  /**
+   * Appends the condition to the given string if the condition is not empty. The condition is added
+   * at the end of the given string in the following form: " ==&gt; condition".
+   *
+   * @param stringRepresentation a string representing this BlockTag
+   * @return the string representation of this tag with the condition appended
+   */
+  String appendCondition(String stringRepresentation) {
```

Comment:
The method name contains "condition" but the field it uses is named "specification".  Be consistent, and consistent with terminology used elsewhere in Toradocu.